bob/mkosi.extra/usr/bin/toggle

```diff
@@ -107,8 +107,11 @@ check_delay() {
 kill_established_connections() {
     local old_mode="$1"
     if [ "$old_mode" = "production" ]; then
-        echo "Killing leftover production flows (port $TITAN_STATE_DIFF_PORT_WSS)..."
+        echo "Killing leftover production flows:"
+        echo " - Titan state diff port $TITAN_STATE_DIFF_PORT_WSS"
+        echo " - HTTPS port $HTTPS_PORT (Flashbots protect tx)"
         $CONNTRACK -D -p tcp --dport $TITAN_STATE_DIFF_PORT_WSS 2>/dev/null
+        $CONNTRACK -D -p tcp --dport $HTTPS_PORT 2>/dev/null
```

Comment:
Should we kill connections not by `dport`, but by IP with `-d`?

Seems more straightforward and less error-prone:
```suggestion
        echo " - Titan state diff ($TITAN_BUILDER_IP)"
        echo " - Flashbots protect tx ($FLASHBOTS_TX_IP1, $FLASHBOTS_TX_IP2)"
        $CONNTRACK -D -d $TITAN_BUILDER_IP 2>/dev/null
        $CONNTRACK -D -d $FLASHBOTS_TX_IP1 2>/dev/null
        $CONNTRACK -D -d $FLASHBOTS_TX_IP2 2>/dev/null
```